cosmovisor/go.mod

```diff
@@ -101,7 +107,7 @@ require (
 	github.com/tendermint/btcd v0.1.1 // indirect
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15 // indirect
 	github.com/tendermint/go-amino v0.16.0 // indirect
-	github.com/tendermint/tendermint v0.35.9 // indirect
+	github.com/tendermint/tendermint v0.34.20 // indirect
```

Comment:
ah nevermind I am wrong. We haven't reverted the main branch, this is actually correct.

Reply:
That's true, but this still leads to dependencies errors: try a go mod tidy in ./cosmovisor on main

Reply:
ohh because it imports the sdk. uhhhhhhh that should be removed imo. Then this is avoided. Il create and issue for now

Reply:
What is the issue? I don't think importing the sdk is an issue is it? That way we are sure to have the right plan type for instance. Plus, the Cosmos SDK does not import Cosmovisor, so I don't think having it present in the go.work makes sense anyway.